_posts/2019-10-18-Reply-to-Childhood-Vision.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Vision Of Your Future Self
+date: 2019-10-18 20:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Me]
+image:
+image2:
+author: Valentino Urbano
+---
+
+[Reply to Childhood Vision][1]:
+
+> A clear mental picture appears to me when I try to imagine who I want to be
+
+This sounds like a great writing prompt for a "What to write about" column.
+
+What would YOU image?
+
+[Started as a comment, but was getting too long]
+
+My idea is less abstract, and more material I guess. I share the idea of this scholar aurea though.
```

Comment:
Is this "aurea" a person? Then use upper case.

Reply:
It is NOT. =)